handler/projects.go

```
@@ -0,0 +1,34 @@
+
+func MakeProjectsHandler() ProjectsHandler {
+	return ProjectsHandler{
+		content: "./storage/fixture/projects.json",
+	}
+}
+
+func (h ProjectsHandler) Handle(w baseHttp.ResponseWriter, r *baseHttp.Request) *http.ApiError {
+	fixture, err := os.ReadFile(h.content)
+
+	if err != nil {
+		slog.Error("Error reading projects file: %v", err)
```

golangci-lint (1.64.8)

slog: slog.Error arg "err" should be a string or a slog.Attr (possible missing key or value)

(govet)